maven-java/kalix-java-protobuf-parent/pom.xml

```diff
@@ -36,10 +36,21 @@
 
         <maven.build.timestamp.format>yyyyMMddHHmmss</maven.build.timestamp.format>
         <java.version>21</java.version>
+        <maven.compiler.release>${java.version}</maven.compiler.release>
+    
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <kalix-sdk.version>1.4.1</kalix-sdk.version>
         <akka-grpc.version>2.1.6</akka-grpc.version>
 
+        <!-- plugin versions -->
+        <maven-shade-plugin.version>2.1</maven-shade-plugin.version>
```

Comment:
I got a few build failures related with how the shade plugin is behaving. 

It turns out that for the protobuf sdk, version 2.1 works. That's the version we had before anyway. 

For code-first we need 3.2.5 (latest) with some extra settings because of Spring. 

Later I want to investigate further why protobuf sdk is not working with the latest.